Assets/Scripts/Projectile.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Projectile : MonoBehaviour {
+	public float xVel;
+	public bool isRight;
+	float yVel = 0.0f;
+	Rigidbody2D rb;
+
+	void Start() {
+		rb = GetComponent<Rigidbody2D>();
+	}
+
+	void Update() {
+		if (isRight) {
+			rb.velocity = new Vector2(xVel, yVel);
+		} else {
+			rb.velocity = new Vector2(-1 * xVel, yVel);
+		}
```

Comment:
Does this need to be set each update loop? what happens if it just gets set on instantiation?

We may want this to be a KinematicBody that doesn't have gravity or other physics applied to it.

Reply:
Good point! Let me look into moving it into the start function